source/fundamentals/linq.txt

```diff
@@ -320,6 +320,90 @@ following documents:
    { "date" : ISODate("2012-12-05T00:00:00Z"), "grade" : "A", "score" : 13 }
    { "date" : ISODate("2012-05-17T00:00:00Z"), "grade" : "A", "score" : 11 }
 
+Nested Statements
++++++++++++++++++
+
+You can chain or nest ``Select`` and ``SelectMany`` statements to unwind nested
+arrays. Consider a collection that contains documents with a **new** schema. These
+documents contain a ``restaurants`` field, which holds an array of documents 
+that each represent the ``Restaurant`` class. The documents within the array each have
```

Comment:
S: since the classes model the documents, I think it's more accurate to say the document is "represented by" Restaurants, rather than represent Restaurants
```suggestion
represented by the ``Restaurant`` class. The documents within the array each have
```